docs/usage-limits.md

```diff
@@ -0,0 +1,66 @@
+---
+slug: usage-limits
+append_help_link: true
+hide_title: true
+description: >-
+  Frequently asked questions about Semgrep, comparisons to similar tools,
+  rule licensing, technical support, and more.
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+
+# Usage limits
+
+This section describes usage limits.
+
+:::caution important dates
+* The Semgrep Community tier will be replaced with Team tier on May 23, 2023.
+:::
+
+### What is changing about Semgrep tiers?
+
+We are sunsetting the Community tier and offering our Team tier for free to all users, subject to a usage limit of 10 contributors scanning on private repositories. A contributor is someone who has contributed code to a private repository being scanned by Semgrep.
+
+You can now access the Team tier features of Semgrep Supply Chain, Semgrep Code, and Semgrep Cloud Platform features such as SSO. Our highest level of support will still be available through the Enterprise tier.
+
+If the number of contributors has exceeded the usage limit, you will need to purchase licensing for the 11th contributor onward. 
+
+### What is the usage limit?
+
+The usage limit is 10 contributors. A contributor is someone who has made at least one commit to a scanned private repository within the last 30 days.
+
+### How are contributors calculated?
+
+Contributors are calculated using `git log` over the past 30 days.
+
+### What happens when the usage limit is exceeded?
+
+Semgrep scans stop when the usage limit is exceeded. You can resume scanning through the following:
+
+* A one-time 30-day free trial (automatically triggered when the usage limit is exceeded for the first time)
+* By purchasing additional licenses
+* By waiting for the next billing cycle
+
+### What if the first scan exceeds the contributor usage limit?
+
+Semgrep will complete the first scan and a one-time 30-day free trial will automatically start. After the trial concludes, if scans are run on private repositories that exceed the usage limit, scans will not run until additional licenses are purchased.
+
+### How is the cost calculated for the Team tier?
+
+The 11th contributor and onwards are charged. An engineering team with 20 contributing developers will only pay for 10 licenses.
```

Comment:
The 11th contributor and onwards are charged. For example, an engineering team...